spec/unit/pops/loaders/loaders_spec.rb

```diff
@@ -246,8 +246,8 @@ def expect_loader_hierarchy(loaders, expected_loaders)
     end
 
     it 'loader allows loading a function more than once' do
-      pending('See PUP-11754')
-      allow(File).to receive(:read).with(user_metadata_path, {:encoding => 'utf-8'}).and_return('')
+      #pending('See PUP-11754')
+      allow(File).to receive(:read).with(user_metadata_path, {:encoding => 'utf-8'}).and_return('{}')
```

Comment:
I'd drop the `pending` and have the test continue to return empty string (since the source code should handle that correctly now when strict mode is enabled)